Lib/test/test_inspect/test_inspect.py

```diff
@@ -5720,7 +5720,10 @@ def test_gc_module_has_signatures(self):
 
     def test_io_module_has_signatures(self):
         methods_no_signature = {
-            'BufferedRWPair': {'read', 'peek', 'read1', 'readinto', 'readinto1', 'write'},
+            'BufferedRWPair': {
+                'read', 'peek', 'read1', 'readinto', 'readinto1', 'write',
+                'backread', 'backreadinto',
+            },
```

Comment:
```suggestion
            'BufferedRWPair': {'read', 'peek', 'read1', 'readinto', 'readinto1', 'write'},
```

Reply:
Thanks! I forgot about them

Reply:
I don't really like this test. I had to modify it recently too.